test/products/searchProductsIT.js

```diff
@@ -144,6 +144,25 @@ describe('magento searchProducts', function() {
                 });
         });
 
+        it('returns a valid response with 2 SKUs', function() {
```

Comment:
Should we add an additional test with more complex filter? Like 2 sku + 1 category and 2 sku + text = shirt.

Reply:
I added an extra test for sku + text, note that in this case the search performs an AND operation.